rules/linux/privilege_escalation_awk_binary_shell.toml

```diff
@@ -0,0 +1,76 @@
+[metadata]
+creation_date = "2022/02/24"
+maturity = "production"
+updated_date = "2022/02/24"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary mawk or nawk abuse to breakout out of restricted shells or environments by spawning an
```

Comment:
Could probably expand on what the commands are normally used for and why this could be malicious